bluemira/codes/_freecadapi.py

```diff
@@ -73,6 +73,29 @@
 WORKING_PRECISION = 1e-5
 MIN_PRECISION = 1e-5
 MAX_PRECISION = 1e-5
+
+# ======================================================================================
+# Error catching
+# ======================================================================================
+
+
+def catch_caderr(raise_error):
+    """
+    Catch CAD errors with given error
+    """
+
+    def argswrap(func):
+        def wrapper(*args, **kwargs):
+            try:
+                return func(*args, **kwargs)
+            except FreeCADError as fe:
+                raise raise_error(fe.args[0]) from None
```

Comment:
If we raise from `None`, do we lose the information the `FreeCADError` is telling us? If so, I think we should still raise from the `FreeCADError`. The issue was basically just saying the error needs to be catchable via `except BluemiraError`